Sources/Brave/Frontend/Shields/SubmitReportView.swift

```diff
@@ -0,0 +1,95 @@
+// Copyright 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import Strings
+import BraveUI
+import DesignSystem
+
+struct SubmitReportView: View {
+  let url: URL
+  let dismiss: () -> Void
+  let submit: (URL, String, String) -> Void
+  
+  @State private var additionalDetails = ""
+  @State private var contactDetails = ""
+  
+  private var scrollContent: some View {
+    ScrollView {
```

Comment:
Also just noticed but you're missing a proper brave background color on this view

Reply:
Done